Neos.ContentRepository.Core/Classes/Projection/CatchUp.php

```diff
@@ -104,7 +104,11 @@ public function run(EventStreamInterface $eventStream): SequenceNumber
                 if ($eventEnvelope->sequenceNumber->value <= $highestAppliedSequenceNumber->value) {
                     continue;
                 }
-                ($this->eventHandler)($eventEnvelope);
+                try {
+                    ($this->eventHandler)($eventEnvelope);
+                } catch (\Exception $e) {
```

Comment:
```suggestion
                } catch (\Throwable $e) {
```

Reply:
In this case i deliberately only handle `Exceptions` as they are somewhat more planned (like the `InvalidArgumentException` from the event normaliser. `Errors` should just not happen (a sight `TypeErrors` i guess but still) and a php type error should just bubble up all the way. If you have a better philosophy im open for it ;)

Reply:
Makes sense